requirements/pip-tools.in

```diff
@@ -0,0 +1,4 @@
+# Dependencies to run compile tools
+-c constraints.txt
+
+pip-tools                           # Contains pip-compile, used to generate pip requirements files
```

Comment:
What is `requirements files` here. Is it some typo mistake...?

Reply:
It refers to the usage of the `pip-compile` command i.e. generation of requirement files using make upgrade job.
If it is confusing we can remove the comment.

Reply:
Ahh I see. So its the comment extending to next line. I though it was listed as a requirement. Nvm it should be good to go

Reply:
nit: new line at the end of file